Engine/SessionLogs.cs

```diff
@@ -73,6 +74,17 @@ public static void Initialize()
         /// </summary>
         public static void Initialize(string tempLogFileName)
         {
+            // We can't just add this as a parameter with a default value because
+            // it isn't backwards compatible with plugins compiled against older versions.
+            Initialize(tempLogFileName, canDeleteLog);
```

Comment:
I dont understand this comment. Why is it not compatible with older plugins?

Reply:
Because on the IL level that would mean removing this overload and adding a method with a different signature